cmd/jujud-controller/agent/machine/manifolds.go

```diff
@@ -1116,7 +1116,7 @@ func CAASManifolds(config ManifoldsConfig) dependency.Manifolds {
 			HTTPClientName:          httpClientName,
 			BootstrapGateName:       isBootstrapGateName,
 			RequiresBootstrap:       bootstrap.RequiresBootstrap,
-			PopulateControllerCharm: bootstrap.PopulateControllerCharm,
+			PopulateControllerCharm: bootstrap.PopulateCAASControllerCharm,
```

Comment:
Maybe the rename needs pushing down into here as well?

Reply:
I may take another stab at this in the future. For now, I these cuts are deep enough.